docs/design.md

```diff
@@ -193,10 +195,10 @@ spec:
   image:
     upload: {} # This is how the plugin signals it wants to upload a directory for building.
   model: # Mounts the model. Plugin auto-populated this by finding the corresponding `model.yaml` file.
-    name: falcon-7b  
-  resources: {...} 
+    name: falcon-7b
+  resources: {...}
 status:
-  uploadUrl: https://some-signed-url... # Controller populated this.
+  uploadURL: https://some-signed-url... # Controller populated this.
```

Comment:
Aligning with the comment here: https://github.com/substratusai/substratus/pull/105#discussion_r1264922505

Just confirming the direction we want here. @samos123 , @nstogner

Reply:
Yes, `URL` should be caps.